packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -440,7 +446,7 @@ const renderPolygonGraphControls = (props: {
                 width: "100%",
                 marginLeft: "20px",
             }}
-            tabIndex={0}
+            tabIndex={coords.length < 3 ? -1 : 0}
```

Comment:
Can you add a comment explaining why the tab index depends on the coords length being less than 3?